pymaker/auctions.py

```diff
@@ -313,19 +323,6 @@ def bids(self, id: int) -> Bid:
                            gal=Address(array[6]),
                            tab=Rad(array[7]))
 
-    def kick(self, usr: Address, gal: Address, tab: Rad, lot: Wad, bid: Rad) -> Transact:
```

Comment:
Do these `kick` methods need to be added to the abstract `DealableAuctionContract` class to support auctions during the migration period?

Reply:
Those contract methods shouldn't have been introduced there in the first place.  `flap` and `flop` auctions are kicked from the `Vow` contract, `flip` auctions from the `Cat` and (now) `clip` auctions from the `Dog`.

Reply:
I see, yeah that makes sense